erfa_generator.py

```diff
@@ -607,7 +607,7 @@ def xfail(self):
                 # Works on newer numpy
                 return "np.__version__ < '1.21', reason='needs numpy >= 1.21'"
             else:
-                return "reason='do not yet support no-input ufuncs'"
+                return "np.__version__ < '1.24', reason='do not yet support no-input ufuncs'"
```

Comment:
Great! Could you also change the docstring?

Reply:
assume you mean the `reason` ... done.